dataland-frontend/tests/component/components/resources/dataTable/MultiLayerDataTableWithEuTaxonomyNonFinancials.cy.ts

```diff
@@ -91,28 +91,26 @@ describe("Component test for the Eu-Taxonomy-Non-Financials view page", () => {
         2,
       );
 
-      getSectionHead("Revenue").click();
+      getSectionHead("Revenue");
```

Comment:
can be deleted

Reply:
.should("exist") added - see comments below